From 21a27c633508b21f90634be8c80205e69c06a555 Mon Sep 17 00:00:00 2001
From: Peacock365 <37863161+Peacock365@users.noreply.github.com>
Date: Sun, 8 Sep 2019 11:38:09 +0200
Subject: [PATCH] (WF68) Silence Crash Reporter.

Disable including the crash URL in crash reports, disable crash reporter for the Flash plugin specifically.
---
 modules/libpref/init/all.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules/libpref/init/all.js b/modules/libpref/init/all.js
index 83a6d60fc72f..07fb44134413 100644
--- a/modules/libpref/init/all.js
+++ b/modules/libpref/init/all.js
@@ -3321,8 +3321,8 @@ pref("dom.ipc.tabs.createKillHardCrashReports", true);
 
 pref("dom.ipc.plugins.flash.disable-protected-mode", false);
 
-pref("dom.ipc.plugins.flash.subprocess.crashreporter.enabled", true);
-pref("dom.ipc.plugins.reportCrashURL", true);
+pref("dom.ipc.plugins.flash.subprocess.crashreporter.enabled", false, locked);
+pref("dom.ipc.plugins.reportCrashURL", false, locked);
 
 // How long we wait before unloading an idle plugin process.
 // Defaults to 30 seconds.
